site/Cargo.toml

```diff
@@ -46,6 +46,7 @@ rust-embed = { version = "6.6.0", features = ["include-exclude", "interpolate-fo
 humansize = "2"
 lru = "0.12.0"
 ruzstd = "0.7.0"
+flate2 = "1.0.32"
```

Comment:
We already use `flate2` in the collector, with a feature set to use the Rust backend (which is however used by default anyway, I think). Let's put the `flate2` dependency line into `workspace.dependencies` of the root `Cargo.toml` file and use `flate2 = { workspace = true }` both here and in `collector`.

I did this for most other shared dependencies in https://github.com/rust-lang/rustc-perf/pull/1961.